_data/jobs.yml

```diff
@@ -1,6 +1,6 @@
 - expires: 2021-08-10
   location: San Diego, CA or remote in US
```

Comment:
```suggestion
  location: Scripps Research, San Diego, CA or remote in US
```

Reply:
Typically jobs show the institution as well as the physical location

Reply:
even better, thanks!